Mathlib/Data/Rat/Cast/Order.lean

```diff
@@ -46,6 +46,9 @@ def castOrderEmbedding : ℚ ↪o K :=
 
 @[simp, norm_cast] lemma cast_lt : (p : K) < q ↔ p < q := cast_strictMono.lt_iff_lt
 
+@[gcongr] alias ⟨_, _root_.GCongr.ratCast_le_ratCast⟩ := cast_le
+@[gcongr] alias ⟨_, _root_.GCongr.ratCast_lt_ratCast⟩ := cast_lt
```

Comment:
Ideally changes to the `gcongr` set would be accompanied by additions to the `gcongr` test suite, but I appreciate it is not always easy to write simple examples, if these were extracted from a bigger branch.